tests/ctest/test_ks.c

```diff
@@ -84,12 +84,12 @@ static void test_creatingLookup (void)
 	KeySet * ks = ksNew (10, KS_END);
 
 	Key * searchKey = keyNew ("user:/something", KEY_VALUE, "a value", KEY_END);
-	Key * k0 = ksLookup (ks, searchKey, KDB_O_CREATE);
```

Comment:
This test should be removed completely. It was testing the functionality of `KDB_O_CREATE`.

The current changes make the test fail (correctly).